.gitignore

```diff
@@ -17,11 +17,7 @@ charts/**/charts
 Chart.lock
 
 # local env files
-local.*
-local-*.*
-.env.local
-.env.*.local
-idplist-local.*
+*local.*
```

Comment:
This isn't completely accurate as the `*local.*` pattern doesn't match situations where we have a `local-`. We should retain most of the original definitions and have the following instead as it'll cover more and be more precise:
```txt
*local.*
*local-*.*
.env.local
.env.*.local
```